scripts/ci/libs.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# be stricter
+set -eou pipefail
+
+# Get librustzcash libraries used in uniffi-zcash-lib
+#
+# Takes the following function args:
+# $1 - librustzcash Cargo.toml path
+# $2 - uniffi-zcash-lib Cargo.toml path
+#
+# Returns:
+# - The labels that are used when searching for or creating a Github issue, in format 'lib_name-current_ver-latest_ver'
+get_libs() {
+	local librustzcash_cargo_path=$1
+	local uniffi_cargo_path=$2
+	if [[ -z "$librustzcash_cargo_path" || -z "$uniffi_cargo_path" ]]; then
+		echo "required parameter for get_libs() is empty" 1>&2
+		exit 1
+	fi
+
+	local output
+	output=$(
+		cargo metadata --format-version=1 --no-deps --quiet --manifest-path="$librustzcash_cargo_path" |
+			jq -r '.packages[] | .name' |
+			xargs -I {} sh -c "cargo metadata --quiet --format-version=1 --no-deps --manifest-path=$uniffi_cargo_path | jq -r '.packages[] | .dependencies[] | .name' | grep '{}' | sort -u | tr '\n' ';'"
+	)
+
+	echo "$output"
+}
+
+# Use jq to get the outdated libs from the "cargo outdated" JSON string
+#
+# Takes the following function args:
+# $1 - "cargo outdated" JSON string
+#
+# Returns:
+# - outdated uniffi librustzcash dependency where the version is not latest, in format - "crate_name;..."
+get_outdated_libs() {
+	local outdated_libs_json=$1
+	if [[ -z "$outdated_libs_json" ]]; then
+		echo "required parameter for get_outdated_libs() is empty" 1>&2
+		exit 1
+	fi
+
+	echo "$outdated_libs_json" | jq -r 'select(.crate_name | startswith("uniffi-") or .=="zcash").dependencies[] | select(.project != .latest) | .name' | sort -u | tr '\n' ';'
```

Comment:
btw I think you're doing this `tr '\n' ';'` pattern and then you do the `IFS=';' read -ar ....`, wouldn't it work without both `tr` and `IFS`?

Reply:
I tried all variants I could think of/find. None of them worked (even the one you suggested). Only the old version worked:

```
IFS=';' read -ra arr <<<"$used_libs"
for lib_name in "${arr[@]}"; do
...
```